Mesh_3/doc/Mesh_3/CGAL/Polyhedral_mesh_domain_with_features_3.h

```diff
@@ -58,7 +58,7 @@ template <typename Polyhedron>
 Polyhedral_mesh_domain_with_features_3(Polyhedron bounding_polyhedron); 
 
 /*!
-Constructs a `Polyhedral_mesh_domain_with_features_3` from an off file. No feature 
+\deprecated Constructs a `Polyhedral_mesh_domain_with_features_3` from an off file. No feature 
```

Comment:
I think a deprecated method:
 -  should have extra documentation saying by what it should be replaced,
 - a mention in the `changes.html`.